electrumx/server/session.py

```diff
@@ -1676,3 +1676,137 @@ def truncate_auxpow(self, headers_full_hex, start_height):
             height += 1
 
         return headers.hex()
+
+
```

Comment:
Similarly here - this is common to all dash derivatives so it should be DashElectrumXBase and have DashElectrumX inherit it.  Then there's no copy-paste.